src/content/en/updates/2017/03/custom-scrollbar.md

```diff
@@ -0,0 +1,380 @@
+      height: 500px;
+    }
+    </style>
+
+And now
+[scroll the box](https://googlechrome.github.io/ui-element-samples/custom-scrollbar/step-4.html)!
+The red box moves down.
+
+## Step 3: Give it a size
+We have an element that moves down when the page scrolls down. That’s the hard
+bit out of the way, really. Now we need to style it to look like a scrollbar and
+make it a bit more interactive.
+
+A scrollbar usually consists of a “thumb” and a “track”, while the track isn’t
+always visible. The thumb’s height is directly proportional to how much of the
+content is visible.
+
+    <script>
+      const scroller = document.querySelector('.container');
+      const thumb = document.querySelector('.box');
+      const scrollerHeight = scroller.getBoundingClientRect().height;
+      thumb.style.height = /* ??? */;
+    </script>
+
+`scrollerHeight` is the height of the scrollable element, while
+`scroller.scrollHeight` is the total height of the scrollable content.
+`scrollerHeight/scroller.scrollHeight` is the fraction of the content that is
+visible. The ratio of vertical space the thumb covers should be equal to the
+ratio of content that is visible:
+
+<img src="/web/updates/images/2017/03/custom-scrollbar/thumbmath01.svg"
+  alt="thumb dot style dot height over scrollerHeight equals scroller height
+  over scroller dot scroll height if and only if thumb dot style dot height
+  equals scroller height times scroller height over scroller dot scroll
+  height.">
+
+    <script>
+      // …
+      thumb.style.height =
+        scrollerHeight * scrollerHeight / scroller.scrollHeight + 'px';
+      // Accommodate for native scrollbars
+      thumb.style.right =
+        (scroller.clientWidth - scroller.getBoundingClientRect().width) + 'px';
+    </script>
+
+Note: We need to adjust `right` so our custom scrollbar is visible on systems
+with permanent native scrollbars. We will completely hide the native scrollbars
+later with a trick.
+
+The size of the thumb is
+[looking good](https://googlechrome.github.io/ui-element-samples/custom-scrollbar/step-5.html),
+but it’s moving way too fast. This is where we can grab our technique from the
+parallax scroller. If we move the element further back it will move slower while
+scrolling. We can correct the size by scaling it up. But how much should we push
+it back exactly? Let’s do some – you guessed it – math! This is the last time, I
+promise.
+
+The crucial bit of information is that we want the bottom edge of the thumb to
+line up with the bottom edge of the scrollable element when scrolled all the way
+down. In other words: If we have scrolled
+`scroller.scrollHeight - scroller.height` pixels, we want our thumb to be
+translated by `scroller.height - thumb.height`. For every pixel scroller, we
+want our thumb should move a fraction of a pixel:
+
+<img src="/web/updates/images/2017/03/custom-scrollbar/thumbmath02.svg"
+  alt="factor equals scroller dot height minus thumb dot height over scroller
```

Comment:
Capitalize 'factor'. Make sure all of your `alt` attributes start with a capital letter.